app/assets/stylesheets/header.css.scss

```diff
@@ -0,0 +1,115 @@
+header {
+  min-height: 50px;
+}
+
+h1 {
+  color: $dark-blue;
+  font-size: 3em;
+  font-weight: bold;
+  text-align: center;
+}
+
+h2 {
+  color: $dark-blue;
+  font-size: 2em;
+  font-style: italic;
+  font-weight: bold;
+}
+
+.navigation-bar {
+  background: $transparent-blue;
+  border-style: ridge none;
+  min-height: 50px;
+  position: relative;
+}
+
+.nav {
+  font-weight: bold;
+  left: 30%;
+  list-style-type: none;
+  margin-top: 0;
+  position: absolute;
+  width: 70%;
+
+  > li {
+    display: inline-block;
+    padding: 15px;
+  }
+
+  a {
+    text-decoration: none;
+   
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+
+.dropdown {
+  background: $white;
+  display: none;
+  list-style: none;
+  padding-left: 0;
+  position: absolute;
+
+  li {
+    padding: 5px 0;
+  }
+    
+  a {
+      white-space: nowrap;
+  }
+}
+
+li:hover ul {
+  background: $transparent-blue;
+  display: block;
+  margin-top: 10px;
+  position: absolute;
+  width: 100px;
+}
+
+
+.logo {
+  border-style: outset;
+  border-width: thick;
+  height: 200px;
+  margin-left: 50px;
+  margin-top: 15px;
+  width: 250px;
+}
+
+.banner-container {
+  position: relative;
+}
+
+.banner {
+  background: $white;
+  height: 250px;
+  overflow: hidden;
+}
+
+.banner-image {
+  bottom: 0;
+  height: 250px;
+  left: 0;
+  margin: auto;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 669px;
+}
+
+.gradient {
+  @include linear-gradient(to right, rgba(255, 255, 255, 1) 0%, 
+    rgba(255, 255, 255, 0) 5%, rgba(255, 255, 255, 0) 95%, 
+    rgba(255, 255, 255, 1) 100%, $fallback: transparent);
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.